packages/checkout/widgets-lib/src/widgets/add-funds/views/AddFunds.tsx

```diff
@@ -614,12 +614,11 @@ export function AddFunds({
                 placeholder="0"
                 maxTextSize="xLarge"
               />
-              {selectedAmountUsd > 0 && (
-                <HeroFormControl.Caption>
-                  USD $
-                  {selectedAmountUsd.toFixed(2)}
-                </HeroFormControl.Caption>
-              )}
+
```

Comment:
@jhesgodi - Tony wants this text to always be there, even the `selectedAmountUsd` is zero. Is that ok?

Reply:
I think that is okay